.github/workflows/main.yml

```diff
@@ -126,56 +113,129 @@ jobs:
         run: yarn
       - working-directory: ./web
         run: npm test
-      - uses: codecov/codecov-action@v1
+      - uses: codecov/codecov-action@a1ed4b322b4b38cb846afb5a0ebfa17086917d27
```

Comment:
pinning is nice - but who is checking for updates if they fix a critical security issue?
for python deps we have requires.io/dependabot, I'm not aware of similar tooling for github action steps? (next startup idea?)

Reply:
Dependabot apparently will notify you if you have it enabled. I haven't tested it.